Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM alpine:3.5
+FROM plugins/base:amd64
+MAINTAINER Drone.IO Community <drone-dev@googlegroups.com>
```

Comment:
Maintainer tag is deprecated, should be replaced with a label if you want one. https://docs.docker.com/engine/reference/builder/#maintainer-deprecated